### ISSUE TITLE
Unable to perform Generate Code operation for requests that are already saved in the collections

### ISSUE THREAD

**Issue by vyskh2498** [usebruno/bruno#2497]
### I have checked the following:

- [X] I use the newest version of bruno.
- [X] I've searched existing issues and found nothing related to my issue.

### Describe the bug

While trying to perform the generate code operation to convert the saved request as a curl or any other code - the operation is not working and taking to the something went wrong page
![Screenshot from 2024-06-21 20-07-05](https://github.com/usebruno/bruno/assets/35701490/bc51628a-07bb-42e9-907e-fe489da703f2)

### Screenshots/Live demo link

![image](https://github.com/usebruno/bruno/assets/35701490/eea46a26-e4f5-45c2-a632-fcb398e0558e)

**Comment by pedrohaccorsi** [comment]
same here. 

Bruno v1.19.0
MacBook Pro, chip M1 Pro, memory 16GB, OS Version 14.5 (23F79)

**Comment by Hyp3rSoniX** [comment]
Yeah, I get this in the console as an error when the error page shows up
```
framework-80ea8c0f440c6a32.js:9 TypeError: Cannot read properties of undefined (reading 'externalSecrets')
    at getExternalCollectionSecretsForActiveEnvironment (_app-92e2050d0799eab9.js:17:186595)
    at getAllVariables (_app-92e2050d0799eab9.js:17:186042)
    at CodeEditor.addOverlay (443-87262752453bea5b.js:1:100581)
    at CodeEditor.value (443-87262752453bea5b.js:1:104274)
    at lk (framework-80ea8c0f440c6a32.js:9:90626)
    at jk (framework-80ea8c0f440c6a32.js:9:111105)
    at framework-80ea8c0f440c6a32.js:9:111116
    at Qk (framework-80ea8c0f440c6a32.js:9:111628)
    at Fk (framework-80ea8c0f440c6a32.js:9:96297)
    at jg (framework-80ea8c0f440c6a32.js:9:44900)
```

But not sure if this is the cause of the issue, or an "ignorable error" popping up when clicking on "Generate Code".

**Comment by jwetzell** [comment]
duplicate of #2444

**Comment by jwetzell** [comment]
This has been resolved with the merge of #2501

**Comment by sanjai0py** [comment]
Closing this as the issue has been fixed in [v1.20.0](https://github.com/usebruno/bruno/releases/tag/v1.20.0).
